src/index.js

```diff
@@ -0,0 +1,273 @@
+/* eslint no-use-before-define: 0, no-extra-parens: 0 */
```

Comment:
nah. by the time anything is called they are actually defined. can expound more when I'm on my computer proper again.